team-config.yml

```diff
@@ -351,4 +351,14 @@ arm:
     build_notices_channel: ""
   tags:
     application: "archiving"
+    businessArea: "Cross-Cutting"
+
+enterprise:
+  team: "platform operations"
+  namespace: ""
+  slack:
+    contact_channel: "#platops-help"
+    build_notices_channel: "#platops-build-notices"
+  tags:
+    application: 'core'
     businessArea: "Cross-Cutting"
```

Comment:
this is an interesting one because the subscriptions will be in multiple areas, probably need to handle this, I see the old bootstrap handled this somehow

Reply:
Updated it to use CFT and SDS. Will have a look to see if there is a way we can use the existing ones for cft-platform and sds-platform by appending `-platform` to my expression https://github.com/hmcts/azure-enterprise/pull/9/files#diff-1349bbf917661e3033e15af35fab86007d045c8e733feb18489ee7db6937afc6R35

Environment related resources still currently set to the above but that's consistent with what ops-bootstrap currently does.